s3stream/src/main/java/com/automq/stream/s3/DeltaWALUploadTask.java

```diff
@@ -55,10 +58,18 @@ public class DeltaWALUploadTask {
     private long uploadTimestamp;
     private long commitTimestamp;
     private volatile CommitStreamSetObjectRequest commitStreamSetObjectRequest;
+    private volatile boolean burst = false;
+    private final long id;
+    private final LongAdder rateLimitTime = new LongAdder();
+    private final LongAdder burstBytes = new LongAdder();
```

Comment:
`rateLimitTime` and `burstBytes` have not been read, is this test code?

Reply:
removed.